files/icons/awesome4.toml

```diff
@@ -43,6 +43,7 @@ net_bridge = "\uf0e8" # fa-sitemap
 net_down = "\u2b07"
 net_loopback = "LO"
 net_modem = "\uf095" # fa-phone
+net_strength = "\uf012"
```

Comment:
Just a though: maybe a better name would be `net_cellular`?

Reply:
Done